src/services/print.js

```diff
@@ -619,6 +619,10 @@ ngeo.Print.prototype.encodeVectorStyleStroke_ = function(symbolizer, strokeStyle
     symbolizer.strokeColor = goog.color.rgbArrayToHex(strokeColorRgba);
     symbolizer.strokeOpacity = strokeColorRgba[3];
   }
+  var strokeDashstyle = strokeStyle.getLineDash();
+  if (goog.isDefAndNotNull(strokeDashstyle)) {
```

Comment:
Done. Thanks for the tip.
Why did you decide these last weeks to avoid the use of goog.isDef ?